kendo-ui/index.d.ts

```diff
@@ -668,15 +668,18 @@ declare namespace kendo.data {
 
         constructor(data?: any);
         init(data?: any): void;
+
+        duration(): number;
+        isMilestone(): boolean;
     }
 
     class GanttDependency extends Model {
         static idField: string;
         static fields: DataSourceSchemaModelFields;
 
-        id: any;
-        predecessorId: number;
-        successorId: number;
+        id: string | number | Object;
+        predecessorId: string | number | Object;
+        successorId: string | number | Object;
```

Comment:
Also, it's not useful to have a type `string | number | Object`, since that's equivalent to just `Object` (or `any`).